PurrSoft/PurrSoft.Application/Filtering/VolunteerFilters.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AlbumStore.Application.Queries.VolunteerQueries;
+using PurrSoft.Domain.Entities;
+
+namespace AlbumStore.Application.Filtering;
+
+public static class VolunteerFilters
+{
+    public static IQueryable<Volunteer> ApplyFilter(this IQueryable<Volunteer> volunteersQuery, GetFilteredVolunteersQueries query)
+    {
+        if (!string.IsNullOrEmpty(query.Search))
+        {
+            volunteersQuery = volunteersQuery.Where(v => v.User.FirstName.Contains(query.Search) || v.User.LastName.Contains(query.Search) || v.User.Email.Contains(query.Search));
```

Comment:
again add more line brakes if possible. Code is hard to read for reviewers if you don't .